### PR TITLE
[KeePassXC]  Add limited Yubikey Support

### PR DESCRIPTION
**Pull request by hmica**
## Description

Try to solve  #3668  by adding limited Yubikey Support

We need to press yubikey on search and action.

To be more usable we should use keepassxc cli with open/close  and reuse opened cli but I will need some helop for this to be done.

## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/keepassxc/package.json [hunk 1]
```diff
@@ -48,6 +48,14 @@
       "description": "KeepassXC.app",
       "placeholder": "Select KeepassXC.app",
       "default": "/Applications/KeePassXC.app"
+    },
+    {
+      "name": "yubikey",
+      "type": "textfield",
+      "required": false,
+      "title": "Yubikey Slot",
+      "description": "Yubikey slot and optional serial used to",
```
Comment (zkytech) [inline extensions/keepassxc/package.json:57]: It is recommended to give a specific example value in the description.

Comment (zkytech) [reply extensions/keepassxc/package.json:57]: eg: `1` or `1:1`?

**Review by zkytech — CHANGES_REQUESTED** [review]
I don't have a device to test this feature, please make sure that it has been tested and works fine.

File: extensions/keepassxc/package.json (hunk 1 above)
Comment (hmica) [reply extensions/keepassxc/package.json:57]: > I don't have a device to test this feature, please make sure that it has been tested and works fine.

Hi,

The fact that we need to touch the device each time is a bit cumbersome so I don't thing the PR worth to be merged as it is, the solution were we use the cli to open the database and kept the reference to the opened cli to make research and have an action to close/release cli will be better.

also sometime if the touch has been made to fast or to slow  it create an exception and we need to restart the raycast action. So if  we have the option  to wait until it's successfully opened with good feedback/retry should be considered.

But I thing a good support of keepassXC in Raycast will be very nice.

For now I have sync low critical password to a less secure database so I can I use the plugin as it is.

**Pull request closed without merging**